internal/envoy/bootstrap.go

```diff
@@ -47,6 +47,7 @@ func Bootstrap(c *BootstrapConfig) *bootstrap.Bootstrap {
 						ConfigType: &listener.Filter_Config{
 							Config: &types.Struct{
 								Fields: map[string]*types.Value{
+									"access_log":  accesslog("/dev/stdout"),
```

Comment:
There are two args to manage the accesslog output: `envoy-http-access-log` & `envoy-https-access-log`, can we use those? Also, access log is configured in listener.go, why move to bootstrap?

Reply:
Those access log args only seem to be available to the `serve` config and not the `bootstrap` config. Should I add those arguments to the `bootstrap` config as well? It would probably have to be changed to something like `envoy-contour-access-log` or something along those lines.